src/Languages/Properties/ListProperties.ts

```diff
@@ -13,7 +13,11 @@ export class ListProperties {
      * The name of the list class.
      */
     public className: string;
-
+    /**
+     * Adds two or more lists together
+     */
+     public concat: NativeCallProperties;
+    /**
```

Comment:
Put a blank line before the `/**` and after the member line. That keeps them visually separated & easier to read.